source/claimchains.rst

```diff
@@ -21,6 +21,11 @@ High level overview of the ClaimChain design
 
 ClaimChains store *claims*
 that users make about their keys and their view of others' keys.
+The chain is self-authenticating and encrypted.
+Cryptographic access control is implemented via capabilities.
+This way the chains can be made publicly available in a block store
+without privacy implications.
```

Comment:
for the last sentence id'' rather say:
"In our design, chains (a series of cryptographically linked blocks) are made publicly  available through a block store without incurring privacy problems. " 
because it's neccessary for our design to work at all we can not say "can be made".

Reply:
and also mention that the block service is "self authenticating" because it's self-authenticating: for a given key only one matching block can be returned etc. 

i'd use "block service" or "block storage" consistently also wrt to the below text.  i prefer "block storage" because it sounds more solid than the cloudy "service" :)